backend/controller/timeline/events_cron.go

```diff
@@ -0,0 +1,70 @@
+package timeline
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+
+	ftlencryption "github.com/TBD54566975/ftl/backend/controller/encryption/api"
+	"github.com/TBD54566975/ftl/backend/controller/timeline/internal/sql"
+	"github.com/TBD54566975/ftl/backend/libdal"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/model"
+)
+
+type CronScheduledEvent struct {
+	ID            int64
+	DeploymentKey model.DeploymentKey
+	Verb          schema.Ref
+
+	Time        time.Time
+	ScheduledAt time.Time
+	Schedule    string
+	Error       optional.Option[string]
+}
+
+func (e *CronScheduledEvent) GetID() int64 { return e.ID }
+func (e *CronScheduledEvent) event()       {}
+
+type eventCronScheduledJSON struct {
+	ScheduledAt time.Time               `json:"scheduled_at"`
+	Schedule    string                  `json:"schedule"`
+	Error       optional.Option[string] `json:"error,omitempty"`
+}
+
+func (s *Service) InsertCronScheduledEvent(ctx context.Context, event *CronScheduledEvent) {
```

Comment:
It's a bit of duplication, but we'll probably want a different type for inserting than returning since the inserted type doesn't need an `ID` an easy way around this it so use 2 types like this:

```go

type CronScheduledEvent struct {
	ID                  int64
	CronScheduled
}

type CronScheduled struct {
	DeploymentKey model.DeploymentKey
	Verb          schema.Ref

	Time        time.Time
	ScheduledAt time.Time
	Schedule    string
	Error       optional.Option[string]
}